2011-04-11  Joseph Myers  <joseph@codesourcery.com>

	* ports/sysdeps/arm/dl-tls.h (TLS_DTV_UNALLOCATED): Define.

diff --git a/ports/sysdeps/arm/dl-tls.h b/ports/sysdeps/arm/dl-tls.h
index 02285a7..694d11d 100644
--- a/ports/sysdeps/arm/dl-tls.h
+++ b/ports/sysdeps/arm/dl-tls.h
@@ -27,3 +27,6 @@ typedef struct dl_tls_index
 
 
 extern void *__tls_get_addr (tls_index *ti);
+
+/* Value used for dtv entries for which the allocation is delayed.  */
+#define TLS_DTV_UNALLOCATED	((void *) -1l)
